### PR TITLE
Add documentation reference to command protocol in health checks.

### PR DESCRIPTION
**Pull request by Sodki**

### CODE REVIEW

File: docs/docs/health-checks.md [hunk 1]
```diff
@@ -74,7 +77,7 @@ OR
   like "[0, 0, 0]" for example, and tasks could be started with port environment
   variables like `$PORT1`.
 * `protocol` (Optional. Default: "HTTP"): Protocol of the requests to be
-  performed. One of "HTTP" or "TCP".
+  performed. One of "HTTP", "TCP", or "Command".
```
Comment (kolloch) [inline docs/docs/health-checks.md]: Typo: It should be "COMMAND" instead of "Command".

File: docs/docs/health-checks.md [hunk 2]
```diff
@@ -58,6 +58,9 @@ OR
 
 #### Health check options
 
+* `command`: Command to run in order to determine the health of a task.
```
Comment (kolloch) [inline docs/docs/health-checks.md]: You have to nest a `"value"` attribute inside. Maybe a link to the section in the rest-api.md would be nice.

**Comment by kolloch** [comment]
Thanks for your corrections! I made some comments. I hope they made sense.

**Comment by aquamatthias** [comment]
Hey @Sodki thanks for the contribution. Do you think you can incorporate the suggested changes?

**Comment by kolloch** [comment]
I will close this for now. Please feel very free to reopen if you update this, @Sodki !

**Pull request closed without merging**